packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.vue

```diff
@@ -14,8 +14,17 @@ const props = withDefaults(defineProps<OnyxTooltipProps>(), {
 defineSlots<{
   /**
    * Default slot where the parent content is placed that should open/close the tooltip.
+   *
+   * **Accessibility**: Please ensure that your content is focusable (e.g. the component itself or by adding a tabindex)
```

Comment:
```suggestion
   * **Accessibility**: Please ensure that your content includes at least one focusable element (by using a button or input element)
```
It's better to not promote `tabindex` to users, otherwise they might not use semantic HTML and creating more issues by making random elements focusable.